forward/forward_test.go

```diff
@@ -35,7 +35,7 @@ func TestCloneRequest(t *testing.T) {
 	const rawPath = "/%E4%B8%9C%E6%96%B9Project"
 	const url = "http://zh.moegirl.org" + rawPath
 	rt := mockRT{func(r *http.Request) (*http.Response, error) {
-		assert.Equal(t, rawPath, r.URL.Opaque, "should not alter the path")
+		assert.Equal(t, url, r.URL.String(), "should not alter the path")
```

Comment:
Since we no longer populate Opaque, I'm just making sure that the String form of the URL is correct.